api/src/main/java/com/fit_goal/Notification.java

```diff
@@ -0,0 +1,21 @@
+package com.fit_goal;
+
+import com.fit_goal.enums.Message;
+import com.fit_goal.enums.Subject;
+import lombok.Getter;
+import lombok.Setter;
+
+
+
+@Getter
+@Setter
+public class Notification {
+
+    private Subject subject;
+    private Message message;
+
+    public Notification(Subject subject, Message message) {
```

Comment:
you can use @AllArgsConstructor instead of manually created one

Reply:
refactored